### ISSUE TITLE
Move lodash to dependencies

### ISSUE THREAD

**Issue by leonciokof** [Hacker0x01/react-datepicker#139]
Error in webpack:

```bash
ERROR in ./~/react-datepicker/dist/react-datepicker.min.js
Module not found: Error: Cannot resolve module 'lodash' in /app/node_modules/react-datepicker/dist
 @ ./~/react-datepicker/dist/react-datepicker.min.js 1:117-134
```

**Comment by ljconde** [comment]
Getting this same one..

**Comment by martijnrusschen** [comment]
Fixed by #140

**Comment by martijnrusschen** [comment]
Thanks, will release these changes today.
